### ISSUE TITLE
Displaying over other apps bug

### ISSUE THREAD

**Issue by JakubKermes** [feelfreelinux/octo4a#396]
On Android 6.0.1 and Sony Xperia Z3, the permissions window is seen as an overlay, which makes installation impossible. You have to go to settings and manually set permissions before opening the app. Then, the OctoPrint installation is successful.

**Comment by sharpie7** [comment]
Yes, similar problem on a galaxy s5. The OctoPrint installer first asks for permission to access photos (IIRC) and then puts up an overlay permissions dialog even if the overlay permissions are already granted.

It seems to be a look at this point, but if you retry several times it eventually goes past the overlay permissions step though I am not sure what is different to allow it to progress.

**Comment by alufers** [comment]
Could you post a screenshot of how it looks?

**Comment by sharpie7** [comment]
Low-tech (because "reasons") photos below (in sequence). 

At least on my device it seems if you respond "slowly" to the "Allow to access photos...." dialog then you don't get the "overlay" one, but if you respond quickly then you do get this dialog.

As you can see, the "overlay" dialog appears even if the permission is already granted. There doesn't seem to be a way to go back from the settings screen to continue the installation. It goes back to the start again.


![IMG_4988](https://user-images.githubusercontent.com/9697874/228264231-0b5e04e3-5bc2-4ad4-9952-430612880514.JPG)
![IMG_4989](https://user-images.githubusercontent.com/9697874/228264312-2c94b106-df92-42fe-ad62-8df5a80d7bda.JPG)
![IMG_4990](https://user-images.githubusercontent.com/9697874/228264346-f40261d2-2130-4fd7-ac84-72f14078279b.JPG)
![IMG_4991](https://user-images.githubusercontent.com/9697874/228264369-f7c83b01-4160-4a57-a424-74ffb7e139e3.JPG)
![IMG_4992](https://user-images.githubusercontent.com/9697874/228264387-7043ba3f-ed7b-491b-9ec0-0d5f5f1e63b8.JPG)

**Comment by JakubKermes** [comment]
Exactly the same, the problem seems to be not exactly granting permission, but window redirecting to settings displaying over
